src/wazuh_db/helpers/wdb_agents_helpers.c

```diff
@@ -43,6 +47,10 @@ int wdb_agents_vuln_cve_insert(int id,
     cJSON_AddStringToObject(data_in, "version", version);
     cJSON_AddStringToObject(data_in, "architecture", architecture);
     cJSON_AddStringToObject(data_in, "cve", cve);
+    cJSON_AddStringToObject(data_in, "reference", reference);
+    cJSON_AddStringToObject(data_in, "type", type);
+    cJSON_AddStringToObject(data_in, "status", status);
+    cJSON_AddBoolToObject(data_in, "check_pkg_existance", check_pkg_existance ? 1 : 0);
```

Comment:
How can I parse the boolean after reading the object from the cJSON structure? Or are you recommending to use int instead of boolean? I thought it was simple to use cJSON_IsNumber.